src/nbl/video/CVulkanConnection.h

```diff
@@ -0,0 +1,59 @@
+#ifndef __NBL_C_VULKAN_CONNECTION_H_INCLUDED__
+#define __NBL_C_VULKAN_CONNECTION_H_INCLUDED__
+
```

Comment:
this file goes in the `include/video` not in the src dir

Reply:
why?

Reply:
you may want to give user access to vulkan hadle or something

Reply:
I repeat my question

Reply:
same for OpenGL connections i guess?

Reply:
yep